clients/admin-ui/src/types/api/models/CoreHealthCheck.ts

```diff
@@ -0,0 +1,29 @@
+/* istanbul ignore file */
+/* tslint:disable */
+/* eslint-disable */
+
+/**
+ * Healthcheck schema
+ */
+export type CoreHealthCheck = {
+  webserver: string;
+  version: string;
+  database: CoreHealthCheck.database;
+  cache: CoreHealthCheck.cache;
+  workers_enabled: boolean;
+  workers: Array<string>;
+};
+
+export namespace CoreHealthCheck {
```

Comment:
actually it's also new to me. this was autogenerated by the `openapi:generate` command, and I think it happened because how I typed things in Python (maybe using `Literal` instead of a full on class or enum? I'm not totally sure, but could play around with it). as far as I can tell, the namespace just nests things inside, so that there can be other types called `cache` and whatnot and it won't conflict

Reply:
ahh, I see now. It's sort of like a module. I didn't realize this was one of the autogenerated files